### ISSUE TITLE
Layer1 -> Layer2 Bridge Failing 

### ISSUE THREAD

**Issue by GenericProgramer** [omgnetwork/optimism-v2#213]
REPOST as the other issue was closed without any discussion and disregarded the fact other browsers were tried. See https://github.com/omgnetwork/optimism-v2/issues/166

*This was closed with no discussion or steps to help debug this issue. As noted in the request, I did try this on several browsers including chrome and firefox. I interact with the wallet in Ledger live. 

*I am a developer and I am happy to take steps to try and debug the issue. 

System Configuration: Nano Leder, Metamask
Browsers: Chrome and Brave
Blind Signing Enabled

Other Information:
The wallet was a legacy one created in 2017 on a nano ledger. When I choose it in metamask it says legacy mew, and doesn't show up in the Ledger Live section.

Issue:
I tried to bridge between L1-L2 before the snapshot the transaction just sat waiting for confirmation. There were no errors in the dev console to suggest there was anything wrong. I tried with several browsers and updated the nano ledger etc. I came back to it this morning and the transaction showed as cancelled.

I tried again this morning with no luck. Interestingly I only had one interaction with metamask and nothing popped up on the nano ledger to confirm.

Other steps:
After the snapshot, I created a new wallet on the nano ledger transferred the tokens to that wallet and then managed to successfully bridge (3) interactions with metamask.

I fear this won't allow me to get the Boba airdrop.

**Comment by CAPtheorem** [comment]
Thanks for your report. Sounds like you have already found a solution to this problem?

> After the snapshot, I created a new wallet on the nano ledger transferred the tokens to that wallet and then managed to successfully bridge (3) interactions with metamask.

**Comment by GenericProgramer** [comment]
> Thanks for your report. Sounds like you have already found a solution to this problem?
> 
> > After the snapshot, I created a new wallet on the nano ledger transferred the tokens to that wallet and then managed to successfully bridge (3) interactions with metamask.

The issue is that I can't claim the Bobba network tokens as the wallet that held OMG at the time of the airdrop is the one that has issues bridging.  The issue is still present with this wallet and interacting with the bobba network.

**Comment by shopdropbeta** [comment]
@genericprogrammer Agreed, when is this issue going to get resolved.  I literally cannot access the Boba network.  I have staked and earned the airdrop and cannot access the network that I staked and airdropped for.  Totally annoying and useless?  I simply cannot bridge to L2.  Can the devs look at adding support for XDEFI wallet as I have no means of get the PK from it to use on Metamask.  This would (should) solve my issue

**Comment by GenericProgrammer** [comment]
> @GenericProgrammer Agreed, when is this issue going to get resolved. I literally cannot access the Boba network. I have staked and earned the airdrop and cannot access the network that I staked and airdropped for. Totally annoying and useless? I simply cannot bridge to L2. Can the devs look at adding support for XDEFI wallet as I have no means of get the PK from it to use on Metamask. This would (should) solve my issue

Wrong @

**Comment by shopdropbeta** [comment]
Right, this blockchain is shit.  This is exactly why projects like this will NEVER get mass adoption.  I am not technical but I am more technical than 99% of the world and I cannot get to the bottom of a 'simple' problem.  I staked OMG for BOBA.  Cannot access my airdrop for whatever shit reason it is that I cannot get from L1 to L2.  No one is here to help, clearly no one cares.  I am outta here. I'm selling my OMG, I will never be able to claim the BOBA and I will tell anyone who cares that BOBA is a wank project.  Good luck and fuck you BOBA

**Comment by CAPtheorem** [comment]
@GenericProgramer - the main point is that depending on the circumstances, the gateway can be entirely bypassed to enable an airdrop claim. The only thing that is needed to claim airdrop BOBA is (*1*) an eligible wallet address and (*2*) the corresponding Merkle proof. So for example the claim can be done with a few lines of an ethers script.

**Comment by shopdropbeta** [comment]
@CAPtheorem  Thank you for getting back at last.  Yes, myself and @GenericProgramer are having different issues. For a start, he knows what he is talking about and also understands your response to me.  From my regard, I accessed the Bobanetwork with an XDEFI wallet which allowed me to stake OMG for the Boba airdrop.  I presumed that, once the airdrop occurred, I could come in, flip from L1 to L2 and have access to the L2 Boba airdrop.  Annoyingly, like all airdrops, loads of other people just pumped OMG on a central exchange like Binance, got their airdrop and then fooked off, selling their OMG and led to an obvious tank in the price.  Anyway, that is not my issue.  When you offer me lines of code, I literally have no idea what to do with that?  I am a non-technical chump who is able to use a basic UI to try and interact with that blockchain.

I came to Bobanetwork, I accessed it using my XDEFI wallet.  It shows (showed, I have now unstaked and sent it to all another wallet) my OMG stake.  I cannot ( and could  not) flip between L1 and L2.  From my perspective, I have an airdrop sitting on the L2 network with no ability to access it and no idea what you are saying in order to access it now?

If you can help me understand what you have said above knowing that I don't know very much, I will follow all of your instructions.  You have to treat me like someone who doesn't know a lot I am afraid.  I can follow instruction but I cant follow things that assume I have technical knowledge of programming and code.

Thank you very much!

**Comment by CAPtheorem** [comment]
Ok got it this is super useful information. Seems like the root issue is that you had your OMG in an XDEFI wallet, and XDEFI does not support Boba, and so it's not easy for you to switch chains to Boba, and therefore, you can't claim your L1 airdrop BOBA, which would be hard in any case because you need to use the Boba gateway to initiate the airdrop, and the Boba gateway supports only MetaMask and things connected to MetaMask - such as hardware wallets. 

So there are two things to do - 

(1) migrate that specific XDEFI account to MetaMask and then use gateway.boba.network. This assumes you have the seed phrase for your XDEFI wallet. Just to state the obvious - **you should never under any circumstances give anyone your seed phrase otherwise your funds will be immediately stolen**. The seed phrase allows you to generate your keys and should allow migration to MetaMask. Unfortunately as far as I can tell, XDEFI does not provide any instructions for how to migrate from XDEFI to MetaMask, they only provide instructions to go from MetaMask to XDEFI. I recommend that you contact XDEFI to get their help on getting your seed phrase (with 12 or 24 words) out of their wallet; you can then use MetaMask and then everything will work.

(2) On our end, we can look into XDEFI to see what the options are there.

**Comment by CAPtheorem** [comment]
Good news - XDEFI uses a standard 12 word seed phrase. When you set up the wallet, you should have seen this: 
<img width="554" alt="Screen Shot 2022-02-13 at 7 12 57 PM" src="https://user-images.githubusercontent.com/79423264/153794098-2f2a2a26-e079-4de0-adaa-82944e1f87de.png">
Using those 12 words, you can create a MetaMask wallet, and then you then use gateway.boba.network to claim your airdrop. If you have lost your seed phrase, that is a much bigger problem anyway, that goes beyond claiming the airdrop.
